### PR TITLE
feat: creates filters for electors page

### PR DESCRIPTION
**Pull request by vivianedias**

### CODE REVIEW

File: pages/_app.js [hunk 1]
```diff
@@ -1,31 +1,40 @@
 import { SessionProvider } from "next-auth/react";
-import { ChakraProvider } from "@chakra-ui/react";
-import { Flex } from "@chakra-ui/react"
-import Header from '../shared/ui/Header/Header'
-import Footer from '../shared/ui/Footer/Footer'
+import { ChakraProvider, extendTheme } from "@chakra-ui/react";
```
Comment (vivianedias) [inline pages/_app.js:2]: pls fix indentation

File: pages/eleitores.jsx [hunk 2]
```diff
@@ -1,19 +1,261 @@
...
+						label={t("filters.lgbt.trans")}
+					/>
+				</Stack>
+			</CheckboxGroup>
+		</Stack>
+	);
+}
+
+function Parties({ t }) {
+	const parties = [
```
Comment (vivianedias) [inline pages/eleitores.jsx]: since this will be a prop coming from api later, lets pass this through props

File: pages/eleitores.jsx [hunk 3]
```diff
@@ -1,19 +1,261 @@
...
+		{ value: "pt", label: "PT" },
+		{ value: "psol", label: "PSOL" },
+		{ value: "pcdob", label: "PCdoB" },
+	];
+	return (
+		<Stack spacing={2}>
+			<Heading as="h2" size="sm" align="left">
+				{t("filters.parties.label")}
+			</Heading>
+			<Field
```
Comment (vivianedias) [inline pages/eleitores.jsx]: looks perfect <3

File: pages/eleitores.jsx [hunk 4]
```diff
@@ -1,19 +1,261 @@
...
+						label={t("filters.priorities.healthEducationWork")}
+					/>
+				</CheckboxGroup>
+			</Flex>
+		</Stack>
+	);
+}
+
+function State({ t }) {
+	const states = [
```
Comment (vivianedias) [inline pages/eleitores.jsx]: lets pass this through the props as well

File: pages/eleitores.jsx [hunk 5]
```diff
@@ -1,19 +1,261 @@
...
+				justifyContent="center"
+				alignItems="center"
+				p="1rem"
+				backgroundColor="whiteAlpha.900"
+				boxShadow="md"
+			>
+				<Form
+					onSubmit={onSubmit}
+					render={({ handleSubmit, submitting, submitError, ...rest }) => {
+						console.log(rest.values);
```
Comment (vivianedias) [inline pages/eleitores.jsx]: lets remove this log and the rest var

File: pages/eleitores.jsx [hunk 6]
```diff
@@ -1,19 +1,261 @@
...
+									<Heading as="h1" size="md" align="left">
+										{t("filters.heading")}
+									</Heading>
+
+									<Identity t={t} />
+									<LGBT t={t} />
+									<Parties t={t} />
+									<Priorities t={t} />
+									<State t={t} />
+
```
Comment (vivianedias) [inline pages/eleitores.jsx]: lets pass this to another comp called "Filters" so we can keep the integration logic in the page, and the rendered comps in the shared folder

File: shared/ui/Eleitores/Eleitores.jsx [hunk 7]
```diff
@@ -0,0 +1,87 @@
+import "/shared/locales/i18n";
```
Comment (vivianedias) [inline shared/ui/Eleitores/Eleitores.jsx]: amazing work in here! Lets just create another folder as `components` and move these files there so we can reuse later

**Review by vivianedias — COMMENTED** [review]
great work on the checkbox fields! super simple solution 💯 🌮

File: shared/ui/Eleitores/Eleitores.jsx (hunk 7 above)
Comment (camilacrdoso) [reply shared/ui/Eleitores/Eleitores.jsx]: thanks ❤️

File: pages/eleitores.jsx (hunk 3 above)
Comment (camilacrdoso) [reply pages/eleitores.jsx]: 💟

**Pull request merged**